.travis.yml

```diff
@@ -45,7 +45,7 @@ jobs:
     - go: "1.12.x"
       os: windows
       # Only run Windows build job on master; it's very slow.
-      if: branch = master AND type = push
+      #if: branch = master AND type = push
```

Comment:
Left over from testing?

Reply:
Not yet :-) Waiting for a Windows Travis run

Reply:
Removed it now since this test passes on Windows. Travis still fails with unrelated stuff:

```
go: writing go.mod cache: rename C:\Users\travis\gopath\pkg\mod\cache\download\github.com\prometheus\common\@v\v0.0.0-20181113130724-41aa239b4cce.mod935043493.tmp C:\Users\travis\gopath\pkg\mod\cache\download\github.com\prometheus\common\@v\v0.0.0-20181113130724-41aa239b4cce.mod: Access is denied.
```